### DISCUSSION TITLE
Docs: Differentiate between snippets and primitives

### DISCUSSION THREAD

**Discussion by veloware** [chakra-ui/chakra-ui#9143] [Ideas]
Would it be useful to in some way make a distinction between components that get added as snippets vs imported directly from Chakra (I think 'primitive' has been the term being used?)

I find that for every component I'm using I have to refer to the docs to see if the component is used via installing as a snippet or via importing directly from Chakra. (I'm having to refer to the docs usually because my IDE will still attempt to import the component directly from Chakra even though it should probably be added as snippet)

Take for example two similar components - Spinner v.s Progress

Spinner 
```
import { Spinner } from "@chakra-ui/react"
```

Progress
```
npx @chakra-ui/cli snippet add progress
```

I don't know what the solution is, or if maybe it's just my lack of understanding of what makes a component a snippet vs a primitive?

Another example, slightly different issue - Say I need a component that looks like a Card, but behaves like a Checkbox, therefore CheckboxCard looks like a good fit, but I may have a few different use cases whereby they have a slightly different DOM structure, interface or whatever, but generally speaking there are a few differences.  At an initial glance, I could go with the snippet that's created after running `npx @chakra-ui/cli snippet add checkbox-card` but it may not fit my needs for various use cases.  From looking at the source of the snippet, and my understanding, I _could_ create my own versions of this snippet using the individual components that are exported from Chakra instead

_MyCustomCheckboxCard_
```
import { CheckboxCard as ChakraCheckboxCard } from '@chakra-ui/react'
...
<ChakraCheckboxCard.Root>...</ChakraCheckboxCard.Root>
```

but there is no documentation for the individual components, like Root, Content etc other than my IDE via intellisense, or from finding, and digging through the source code.

I think there is a slight gap in the docs around what is a CheckboxCard as a snippet, i.e. a closed component, vs. a CheckboxCard as a (namespaced) component imported from Chakra, and how you should use the latter.

**Comment by reminjp** [comment]
I was also confused by this distinction before.  

To make the difference clearer, the "Components" tab in the documentation could be divided into two sections.  

If Chakra UI's policy doesn't strictly distinguish between them, how about including separate "Snippet" and "Primitive" sections on each component page?

**Comment by segunadebayo** [comment]
Duplicate of https://github.com/chakra-ui/chakra-ui/discussions/9326

Snippets are optional copies of Provider, toaster, and a few composed pieces. Almost every component is exported from `@chakra-ui/react`. Use the primitive if the snippet does not fit.
